models/index.js

```diff
@@ -89,3 +71,17 @@ exports.addCommentById = (newComment, article_id) => {
     return rows;
   });
 };
+
+exports.updateVotesById = (instructions, article_id) => {
+    const { inc_votes } = instructions;
+    if (![-100, 1].includes(inc_votes)) {
```

Comment:
Are you really going to allow your user to upvote by 1...or downvote by 100? That seems extreme and not the best way to handle a bad request. Your psql error handlers should be able to take care of this rather than a custom reject.